### PR TITLE
Issue #180 simplify division and group selections

### LINKED ISSUE #180: Simplify Division and Group selection on Manage teams
As an user I want to have simplified workflow for adding new teams to the tournament so I won't spend a lot of time doing repetitive work.

AC:

- When there is only one group in selector - do not show it
- When there is only one division in selector - do not show it
- even if the selector is not shown correct value should be saved.

- [x] #185 Hide group selector

- [x] #186 Hide Division selector

- [x] #187 Save data

- [x] #207  Test adding new team to tournament

### PR DESCRIPTION
**Pull request by NataliiaBidiuk**
### Summary

We change "activeElements" to getElementsByName('divisions') because in all these methods we use only this element, also we add new variables for set visibility...

closes #180 

### Areas Of Interest

We found some bug with saving information. Need some notifications that Team is not selected
image

#### Design

- [ ] Web API layer contains as little logic as possible
- [ ] Domain objects use semantic names

#### Perfromance

- [ ] In case of any DB query was changed: attach generated SQL for review
- [ ] Number of DB roundtrips should be as low as possible

#### Unit Tests

- [ ] Naming: Initial state and expected result are properly stated
- [ ] Test follows [AAA](https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80) pattern.
- [ ] Test is Isolated
- [ ] Mock instances and expected instances do not share references
- [ ] When new property is added all related comparers are updated

### CODE REVIEW

File: src/VolleyManagement.Backend/VolleyManagement.UI/Scripts/VMScripts/TournamentOperations/AddTeams.js [hunk 1]
```diff
@@ -131,19 +132,29 @@ $(document).ready(function () {
...
+                responseOptions = "<option value = '0'>" + currNs.divisionIsNotSelectedMessage + "</option>";
+                divisionVisibility = "visible";
+            }
 
             $.each(options, function (key, value) {
                 responseOptions += "<option value='" + value.Id + "'>" + value.Name + "</option>";
             });
             privates.renderNewTournamentDivisionsRow(responseOptions);
+
+            if (options.length == 1) {
```
Comment (volleymanagement-sonarqube) [inline src/VolleyManagement.Backend/VolleyManagement.UI/Scripts/VMScripts/TournamentOperations/AddTeams.js]: MAJOR Replace "==" with "===". [rule](https://sonarcloud.io/coding_rules#rule_key=javascript%3AEqEqEq)

**Comment by volleymanagement-sonarqube** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: src/VolleyManagement.Backend/VolleyManagement.UI/Scripts/VMScripts/TournamentOperations/AddTeams.js [hunk 2]
```diff
@@ -6,7 +6,8 @@ $(document).ready(function () {
     var divisionCounter = 0;
     var teamCounter = 0;
     var MAX_TEAMS_NUMBER = 0;
-    
+    var groupVisibility = "hidden";
```
Comment (sdiachen) [inline src/VolleyManagement.Backend/VolleyManagement.UI/Scripts/VMScripts/TournamentOperations/AddTeams.js]: Why don't you use jQuery to control visibility?

File: src/VolleyManagement.Backend/VolleyManagement.UI/Scripts/VMScripts/TournamentOperations/AddTeams.js [hunk 3]
```diff
@@ -131,19 +132,29 @@ $(document).ready(function () {
...
             });
             privates.renderNewTournamentDivisionsRow(responseOptions);
+
+            if (options.length === 1) {
+                privates.CheckIfEmptyGroupRowDraw();
+            }
+
             $(".deleteTeamButton").bind("click", currNs.onDeleteTeamButtonClick);
-            $('select[name="divisions"][counter="' + divisionCounter + '"]').on('change', privates.CheckIfEmptyGroupRowDraw);           
+            $('select[name="divisions"][counter="' + divisionCounter + '"]').bind('change', privates.CheckIfEmptyGroupRowDraw);
```
Comment (sdiachen) [inline src/VolleyManagement.Backend/VolleyManagement.UI/Scripts/VMScripts/TournamentOperations/AddTeams.js]: What was the reason here?

Comment (ViznyakS) [reply src/VolleyManagement.Backend/VolleyManagement.UI/Scripts/VMScripts/TournamentOperations/AddTeams.js]: sorry, it's made by mistake)

File: src/VolleyManagement.Backend/VolleyManagement.UI/Scripts/VMScripts/TournamentOperations/AddTeams.js (hunk 2 above)
Comment (ViznyakS) [reply src/VolleyManagement.Backend/VolleyManagement.UI/Scripts/VMScripts/TournamentOperations/AddTeams.js]: ok, we will try to use jQuery

**Pull request merged**